js/Genoverse.js

```diff
@@ -373,6 +377,7 @@ const Genoverse = Base.extend({
         // Subtracting the difference between them gives you back the correct height to input back into the track when loading configuration
         conf = {
           id         : this.tracks[i].id,
+          defaultId  : this.tracks[i].defaultId,
```

Comment:
Adding defaultId to localhost config. Just to be able load in tracks correctly from library. As id on tracks is manipulated it doesn't find correct tracks anymore.